### DISCUSSION TITLE
How to call Appwrite function from postman?

### DISCUSSION THREAD

**Discussion by chaitanya1006** [appwrite/appwrite#4946] [Q&A] [answered]
Hello all,

Any idea how to call the Appwrite function from the postman?  I have tried the instructions in the documentation but those endpoints are returning function metadata, but not the function response.

I have the appwrite instance installed locally and successfully deployed the function with name '**greetings**'.

Function path: **functions/greetings**

I am calling the deployed function from postman like below with projectID and API Key: 

(GET request)
`http://localhost/v1/functions/greetings`

But, this endpoint is returning the error:

`{
    "message": "Function with the requested ID could not be found.",
    "code": 404,
    "type": "function_not_found",
    "version": "1.2.0"
}`

Any suggestions highly appreciated.

Thank you.

**Comment by stnguyen90** [comment, marked as answer]
To execute the function, you'll need to make a POST request to `http://localhost/v1/functions/{functionId}/executions`. The folder path is the function name and may not actually be the function ID.

FYI, I recommend using HTTPS instead of HTTP in order to avoid any HTTPS redirect problems.

For reference:

* [Function API docs][1]
* [REST API docs][2]

  [1]: https://appwrite.io/docs/client/functions?sdk=web-default#functionsCreateExecution
  [2]: https://appwrite.io/docs/rest

**Comment by ajahuanex** [comment]
**_X-Appwrite-Project_**  is required in header to postman

![image](https://github.com/appwrite/appwrite/assets/2767765/fa7b8a6b-e6bb-4796-b208-16b6095eac5f)

**Comment by fawkeswei** [comment]
How can path be specified when calling from postman? 

Do this `http://localhost/v1/functions/{functionId}/executions/{path}` will result in 404
